--- a/Makefile	2019-01-24 16:57:36.950055238 +0100
+++ b/Makefile	2019-01-24 16:59:53.961284697 +0100
@@ -1,10 +1,12 @@
 install:
-	mkdir -p /usr/share/bootini
-	install boot.ini /usr/share/bootini
-	install c1_init.sh /etc/initramfs-tools/scripts/local-top
-	install 99-c1-network.conf /etc/sysctl.d
-	install update_20150724.sh /usr/share/bootini
-	install 99-gpiomem.rules /etc/udev/rules.d
-	install boot.ini.default /usr/share/bootini
-	install bootini-persistence.pl /usr/share/bootini
-	
\ Kein Zeilenumbruch am Dateiende.
+	mkdir -p $(DESTDIR)/usr/share/bootini
+	install boot.ini $(DESTDIR)/usr/share/bootini
+	mkdir -p $(DESTDIR)/etc/initramfs-tools/scripts/local-top
+	install c1_init.sh $(DESTDIR)/etc/initramfs-tools/scripts/local-top
+	install -D 99-c1-network.conf -t $(DESTDIR)/etc/sysctl.d
+	install update_20150724.sh $(DESTDIR)/usr/share/bootini
+	mkdir -p $(DESTDIR)/etc/udev/rules.d
+	install 99-gpiomem.rules $(DESTDIR)/etc/udev/rules.d
+	install boot.ini.default $(DESTDIR)/usr/share/bootini
+	install bootini-persistence.pl $(DESTDIR)/usr/share/bootini
+
