Index: c1-bootini-0/Makefile
===================================================================
--- c1-bootini-0.orig/Makefile
+++ c1-bootini-0/Makefile
@@ -1,6 +1,7 @@
 install:
-	mkdir -p /usr/share/bootini
-	install boot.ini /usr/share/bootini
-	install c1_init.sh /etc/initramfs-tools/scripts/local-top
-	install 99-c1-network.conf /etc/sysctl.d
-	install update_20150724.sh /usr/share/bootini
+	mkdir -p $(DESTDIR)/usr/share/bootini
+	install boot.ini $(DESTDIR)/usr/share/bootini
+	mkdir -p $(DESTDIR)/etc/initramfs-tools/scripts/local-top
+	install c1_init.sh $(DESTDIR)/etc/initramfs-tools/scripts/local-top
+	install -D 99-c1-network.conf -t $(DESTDIR)/etc/sysctl.d
+	install update_20150724.sh $(DESTDIR)/usr/share/bootini
